monkey/infection_monkey/exploit/tools/helpers.py

```diff
@@ -20,9 +20,9 @@ def get_random_file_suffix() -> str:
 
 def get_agent_dest_path(host: VictimHost, options: Mapping[str, Any]) -> Path:
```

Comment:
Does the type checker not complain about this? 
```suggestion
def get_agent_dest_path(host: VictimHost, options: Mapping[str, Any]) -> PurePath:
```

Reply:
It doesn't complain because `PurePath` is a descendant of `Path`

Reply:
^ not true. Not sure why it doesn't complain